webpack/localization-plugin/src/interfaces.ts

```diff
@@ -153,9 +153,10 @@ export interface ILocalizationPluginOptions {
   localizedData: ILocalizedData;
 
   /**
-   * This option is used to specify `.resx` and `.loc.json` files that should not be processed by this plugin.
+   * This option is used to specify `.resx`, `.resx.json` and `.loc.json` files that should not be processed by
+   * this plugin.
```

Comment:
The PR description gave this example:

```
globsToIgnore: [ 'node_modules/package-name/lib/**/*.loc.json' ]
```

Some questions about the spec:

- Will the crawled paths always be relative to the project folder?
- Will the crawled paths always end with one of the 3 extension `.resx`, `.resx.json` and `.loc.json`?
- Is the glob matcher recursively crawling into the `node_modules` folder?  (How does it handle PNPM symlinks?)

Reply:
- There's no crawl: Webpack encounters references from the input code, and this plugin intercepts them.
- That's currently all the rule matches, yeah.
- See point #1.